### PR TITLE
Add ScholarX 2021 and ScholarX summer to archives

### LINKED ISSUE #1173: Add ScholarX 2021 and ScholarX summer to archives
**Is your feature request related to a problem? Please describe.**
We need to add scholarx 2021 and scholarx summer to the https://sefglobal.org/scholarx/archive/ page

**Describe the solution you'd like**
Create a cards for scholarx 2021 and scholarx summer in https://github.com/sef-global/sef-site/blob/master/scholarx/archive/index.html

**Describe alternatives you've considered**
N/A

**Additional context**
Contact the scholarx team to verify the status of scholarx summer before moving it to archive

**Are you working on it?**
No

### PR DESCRIPTION
**Pull request by YoshithaRathnayake**
## Purpose

The purpose of this PR is to fix #1173 

## Goals

Adding ScholarX 2021 and ScholarX summer to archives

## Approach

I added ScholarX 2021 and ScholarX summer to archives

### Screenshots

image

  
### Preview Link

https://pr-1185-sef-site.surge.sh/scholarx/archive

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: scholarx/archive/index.html [hunk 1]
```diff
@@ -107,6 +114,15 @@ <h5 class="card-title">ScholarX 2019</h5>
                     </p>
                 </div>
             </a>
+            <a href="2021/summer" class="card col-md-6 shadow archive-card">
+                <img src="images/card-image.jpg" class="card-img card-template rounded-top25px" alt="2019">
+                <div class="card-body">
+                    <h5 class="card-title">ScholarX Summer</h5>
```
Comment (anjula-sack) [inline scholarx/archive/index.html]: This should be `ScholarX Summer 2021`

Comment (YoshithaRathnayake) [reply scholarx/archive/index.html]: Ok !

**Comment by YoshithaRathnayake** [comment]
Ok @anjula-sack did you find the text I should put for the scholarx summer 2021

**Comment by YoshithaRathnayake** [comment]
@Piumal1999 is that new 2021-summer file is good? If there is a wrong tell me.

**Review by anjula-sack — CHANGES_REQUESTED** [review]
Can you remove this from the 2021 page?
Screenshot 2022-04-10 at 9 29 15 AM

**Comment by YoshithaRathnayake** [comment]
> Can you remove this from the 2021 page? Screenshot 2022-04-10 at 9 29 15 AM

Now is it ok? @anjula-sack

**Review by anjula-sack — APPROVED** [review]
Good job! @YoshithaRathnayake

**Pull request merged**